addons/l10n_se/data/res_country_data.xml

```diff
@@ -6,7 +6,7 @@
         <field name="priority" eval="900"/>
         <field name="arch" type="xml">
             <form>
-                <div class="o_address_format">
+                <div class="o_address_format" style="margin-top: -23px;">
```

Comment:
i don't think that's the right way to fix this, specially because we try to avoid using inline styling at all cost. Why ? 
- Inline styling will take priority on every class
- It's not robust at all
- readability is not great either

Reply:
Also it seems it's a more general issue in every localisation that did a form view to change the address format. It's because we replace the address block by a form view in here
https://github.com/odoo/odoo/blob/92301a5b300dec1ddfca44dc35318b83d67c56fa/odoo/addons/base/models/res_partner.py#L86

i'm wondering if we could not add a custom class in that function (seems that adding a class to the form directly gets deleted), and do some scss to remove the padding added by the form view (which is the thing that make the address block going a bit below)

Reply:
Since it's a general issues maybe we prefer to not do anything (see with po)